apps/full-site-editing/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@automattic/full-site-editing",
```

Comment:
Not changing this one?
Given that is a private package, changing the name shouldn't have any unexpected repercussion